libs/windy-sounding/src/style/skewt.less

```
@@ -0,0 +1,349 @@
+  // ---------------------------------------------------------------------------
+  // Base Graph Container
+  // ---------------------------------------------------------------------------
+
+  .graph {
+    shape-rendering: geometricprecision;
+
+    // Background canvas & borders
+    .background {
+      fill: white;
+      stroke: none;
+      shape-rendering: crispEdges;
```

Stylelint (17.14.0)

[error] Expected "crispEdges" to be "crispedges" (value-keyword-case)

(value-keyword-case)